src/website/stores/Timestamp/TimestampTypes.ts

```diff
@@ -0,0 +1,9 @@
+export type Timestamp = {
+  startDate: string | null;
+  endDate: string | null;
+};
+
+export type TimestampState = {
```

Comment:
Will need to change this to `DataFilterState`

Reply:
done